instagram/src/components/CommentSection/CommentInput.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InputForm = styled.form`
+    border: 1px solid white;
+    border-top: 1px solid lightgrey;
+    width: 97%;
+    height: 40px;
+    text-align: left;
+    font-size: 15px;
+    color: grey;
+    margin: 5px 10px;
+`;
+
+const CommentInputForm = styled.input`
```

Comment:
Suggestion for comment input to be black when there is something being written. Currently you have it set to the same color as the default grey.

Reply:
I do like that better!